_includes/releases/v22.2/v22.2.0.md

```diff
@@ -0,0 +1,115 @@
+
+With the release of CockroachDB v22.2, we've added new capabilities in CockroachDB to help you build, optimize, and operate more effectively and efficiently. Check out a [summary of the most significant user-facing changes](#v22-2-0-feature-highlights) and then [upgrade to CockroachDB v22.2](../v22.2/upgrade-cockroach-version.html).
+
+For demos and videos on the new features, see the [v22.2 page](https://www.cockroachlabs.com/22-2-launch). [Join our webinar](http://cockroachlabs.com/webinars/maximize-queries-per-second-with-distributedsql) on schema design and query tuning in a distributed SQL database.
+
+{% include releases/release-downloads-docker-image.md release=include.release %}
+
+<h3 id="v22-2-0-{{ site.data.products.db | downcase | replace: " ", "-" }}">{{ site.data.products.db }}</h3>
+
+- <a href="https://cockroachlabs.cloud/signup?referralId=docs_crdb_release_notes" rel="noopener" target="_blank">Get a free v22.2 cluster on {{ site.data.products.serverless }}</a>.
+- Learn about recent updates to {{ site.data.products.db }} in the [{{ site.data.products.db }} Release Notes](cloud.html).
+
+<h3 id="v22-2-0-feature-highlights">Feature highlights</h3>
+
+This section summarizes the most significant user-facing changes in v22.2.0. For a complete list of features and changes, including bug fixes and performance improvements, see the [release notes](index.html#testing-releases) for previous testing releases. You can also search for [what's new in v22.2 in our docs](../search.html?query=new%20in%20v22.2).
+
+{{site.data.alerts.callout_info}}
+The features highlighted below are freely available in {{ site.data.products.core }} and do not require an [enterprise license](https://www.cockroachlabs.com/get-cockroachdb/enterprise/), unless otherwise noted. [{{ site.data.products.db }} clusters](https://cockroachlabs.cloud/) include all enterprise features. You can also use [`cockroach demo`](../v22.2/cockroach-demo.html) to test enterprise features in a local, temporary cluster.
+{{site.data.alerts.end}}
+
+- [Streamline how you build](#v22-2-0-streamline-how-you-build)
+- [Move data between systems](#v22-2-0-move-data-between-systems)
+- [Improve application performance without downtime](#v22-2-0-improve-application-performance-without-downtime)
+- [Protect your data](#v22-2-0-protect-your-data)
+- [Backward-incompatible changes](#v22-2-0-backward-incompatible-changes)
+- [Deprecations](#v22-2-0-deprecations)
+- [Known limitations](#v22-2-0-known-limitations)
+- [Additional resources](#v22-2-0-additional-resources)
+
+<style>
+    table td:first-child {
+        min-width: 100px !important;
+    }
+    table td:nth-child(2) {
+        min-width: 200px !important;
+    }
+</style>
+
+<h4 id="v22-2-0-streamline-how-you-build">Streamline how you build</h4>
+
+ Feature                                                       | Description
+---------------------------------------------------------------+----------------
```

Comment:
Noticed: these headers especially, but also some of the language beneath them, is a marked departure from the v22.1 version of this page. Since you also wrote those RNs, I intuit that this marketing-focused language change was handed down from above, so I'll say nothing more.